android/AndroidManifest.xml

```diff
@@ -557,24 +559,10 @@
         <data android:scheme="https"/>
 
         <!-- #map=$zoom/$lat/$lon -->
-        <data android:host="www.openstreetmap.org" android:path="/"/>
-        <data android:host="www.openstreetmap.org" android:pathPrefix="/search"/>
-        <data android:host="openstreetmap.org" android:path="/"/>
-        <data android:host="openstreetmap.org" android:pathPrefix="/search"/>
-      </intent-filter>
-
-      <intent-filter>
-        <action android:name="android.intent.action.VIEW"/>
-
-        <category android:name="android.intent.category.DEFAULT"/>
-        <category android:name="android.intent.category.BROWSABLE"/>
-
-        <data
-          android:host="lead"
-          android:scheme="mapsme"/>
-        <data
-          android:host="lead"
-          android:scheme="mapswithme"/>
+        <data android:host="openstreetmap.org"/>
+        <data android:host="www.openstreetmap.org"/>
```

Comment:
Are there any shortened OSM URLs we also should handle in a similar way?

Reply:
Yeah, they have a URL shortener, but we don't support it: `https://osm.org/go/2EE6hs--`.

I added this variant to the list: https://github.com/organicmaps/organicmaps/issues/475